src/tasks/DockerBuildTaskProvider.ts

```diff
@@ -32,6 +30,12 @@ export class DockerBuildTaskProvider extends DockerTaskProvider {
 
     // TODO: Skip if image is freshly built
     protected async executeTaskInternal(context: DockerBuildTaskContext, task: DockerBuildTask): Promise<void> {
+
+        if (task.definition.netCore?.useSdkBuild) {

```

Comment:
Any .NET specific behavior should be implemented by overriding the new build function in NetCoreTaskHelper; DockerBuildTaskProvider shouldn't implement any custom behavior for .NET specific build actions.

Reply:
Good catch! I forgot to delete it after I moved the logic to `NetCoreTaskHelper` :(